QualifiedTeachersApi/src/QualifiedTeachersApi/V2/Validators/GetOrCreateTrnRequestValidator.cs

```diff
@@ -220,5 +220,18 @@ public GetOrCreateTrnRequestValidator(IClock clock)
             .When(r => r.TeacherType == CreateTeacherType.OverseasQualifiedTeacher, ApplyConditionTo.CurrentValidator)
             .Null()
             .When(r => r.TeacherType != CreateTeacherType.OverseasQualifiedTeacher, ApplyConditionTo.CurrentValidator);
+
+        RuleFor(r => r.SlugId)
+            .NotEmpty()
```

Comment:
I don't think we want to enforce this as it will make rolling this out difficult.